float64[] joint_value
---
float64[] jacobian
